core-bundle/contao/widgets/ListWizard.php

```diff
@@ -82,8 +82,8 @@ public function generate()
 				}
 				else
 				{
-					$return .= ' <button type="button" data-command="' . $button . '">' . Image::getHtml($button . '.svg', $GLOBALS['TL_LANG']['MSC']['lw_' . $button]) . '</button>';
-				}
+				$return .= ' <button type="button" data-command="' . $button . '">' . Image::getHtml($button . '.svg', $GLOBALS['TL_LANG']['MSC']['lw_' . $button]) . '</button>';
+			}
```

Comment:
```suggestion
					$return .= ' <button type="button" data-command="' . $button . '">' . Image::getHtml($button . '.svg', $GLOBALS['TL_LANG']['MSC']['lw_' . $button]) . '</button>';
				}
```

Unrelated change that should be reverted

Reply:
Fixed in 5960ec1f9982810375a6b64f15c5d81c41c5f869